data/transactions/logic/assembler_test.go

```diff
@@ -1345,6 +1346,10 @@ gtxn 12 Fee
 txn ExtraProgramPages
 txn Nonparticipation
 global CurrentApplicationAddress
+txna Logs 1
+txn NumLogs
+txn EvalConfigAsset
+txn EvalApplicationID
```

Comment:
No, the new itxn fields are actually new txn fields.  Currently they fail when used with with txn, but I think in the future they should become supported, if asking about a previous txn in your group.  It will give a uniform way to access logs and creatable ids.

Reply:
then maybe add itxn to this assembly test as well?

Reply:
Sure, makes sense.  I thought this one was the field test when I added those, but it should have a nice selection of opcodes too.